readme.md

```diff
@@ -103,17 +103,17 @@ For details on OS support see the following resources:
+| [Identity registry (CRUD)](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry) | :heavy_check_mark:        | HTTP | RegistryManager | Use your backend app to perform CRUD operation for individual device or in bulk.                                         ||
+| [Query](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-query-language)                       | :heavy_check_mark:        | HTTP | RegistryManager | Use your backend app to query for information on device twins, module twins, jobs and message routing.                                                                   |
+| [Import/Export jobs](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-jobs)                                  | :heavy_check_mark:        | HTTP | RegistryManager | Use your backend app to import or export device identities in bulk.                                                                      |
+| [Scheduled jobs](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-jobs)                                  | :heavy_check_mark:        | HTTP | JobsClient | Use your backend app to schedule jobs to update desired properties, update tags and invoke direct methods.
+| [Cloud-to-device messaging](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-messages-c2d)     | :heavy_check_mark:        | AMQP | ServiceClient | Use your backend app to send cloud-to-device messages in AMQP and AMQP-WS, and set up notifications for cloud-to-device message delivery. |
+| [Direct Methods operations](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-direct-methods)   | :heavy_check_mark:        | HTTP | ServiceClient | Use your backend app to invoke direct method on device.                                                                  |
+| [File Upload Notifications](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-file-upload)                    | :heavy_check_mark:        | AMQP | ServiceClient | Use your backend app to receive file upload notifications.
+| [IoT Hub Statistics](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-metrics)                          | :heavy_check_mark:        | HTTP | ServiceClient | Use your backend app to get IoT hub identity registry statistics such as total device count for device statistics, and connected device count for service statistics.
+| [Digital Twin Operations](https://docs.microsoft.com/en-us/azure/iot-pnp/overview-iot-plug-and-play)              | :heavy_check_mark:        | HTTP | DigitalTwinClient or RegistryManager | Use your backend app to perform operations on plug and play devices. The operations include get twins, update twins and invoke commands.                                              |
```

Comment:
Since the linked doc here points to pnp, we should guide the users to use the DigitalTwinClient here.

Reply:
True but we need to give both options right? I was going to write that one is preferred but it seemed to wordy for the table.

Reply:
I've added a note in the description in the PR against master directly.